### PR TITLE
Fix tutorial command for getting the pod's ip

### PR DESCRIPTION
**Pull request by arturo-seijas**

### CODE REVIEW

File: docs/tutorial/getting-started.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ juju add-model wordpress-tutorial
 
 ### Deploy wordpress-k8s charm
 
-Deployment of WordPress requires a database integration. The preferred method in juju is through an
-integration. In this case, a `mysql` [interface](https://juju.is/docs/sdk/integration) is required
-by the wordpress-k8s charm and hence, [`mysql-k8s`](https://charmhub.io/mysql-k8s) charm will be
-used.
+Deployment of WordPress requires a database integration. The preferred method to provide it 
+in juju is through a elation. In this case, a `mysql` [interface](https://juju.is/docs/sdk/integration)
```
Comment (mthaddon) [inline docs/tutorial/getting-started.md:31]: We've switched from saying an integration here to a relation (or elation)

File: docs/tutorial/getting-started.md [hunk 2]
```diff
@@ -27,10 +27,10 @@ juju add-model wordpress-tutorial
...
+is required by the wordpress-k8s charm and hence, [`mysql-k8s`](https://charmhub.io/mysql-k8s) 
+charm will be used.
```
Comment (weiiwang01) [inline docs/tutorial/getting-started.md:33]: Hi, could we keep using the name "integration" when referring the juju relation?

File: docs/tutorial/getting-started.md (hunk 1 above)
Comment (arturo-seijas) [reply docs/tutorial/getting-started.md:31]: Ok, so which one of the two "integraiton" appearances should we reword, and how?

File: docs/tutorial/getting-started.md (hunk 2 above)
Comment (arturo-seijas) [reply docs/tutorial/getting-started.md:33]: Ok, so which one of the two "integraiton" appearances should we reword, and how?

**Review by weiiwang01 — APPROVED** [review]

**Review by mthaddon — APPROVED** [review]

**Pull request merged**